src/components/PostDetails.tsx

```diff
@@ -1,106 +1,121 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Comment, CommentData } from '../types/Comment';
+import { Post } from '../types/Post';
+import { User } from '../types/User';
+
+type Props = {
+  showCommentLoader: boolean;
+  postComments: Comment[];
+  selectedPost: Post | null;
+  errorsComment: boolean;
+  handleDeleteComment: (commentId: number) => void;
+  addComment: (newComment: CommentData) => void;
+  buttonAddLoading: boolean;
+  setShowCommentForm: (item: boolean) => void;
+  showCommentForm: boolean;
+  userSelected: User | null;
+};
+
+export const PostDetails: React.FC<Props> = ({
+  showCommentLoader,
+  postComments,
+  selectedPost,
+  errorsComment,
+  handleDeleteComment,
```

Comment:
Your method name `handleDeleteComment` does not follow the recommended naming conventions for event handlers in React. It's better to name it `onDeleteComment` according to this [guide](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364).